src/charm.py

```diff
@@ -740,6 +730,153 @@ def _gen_wp_config(self):
         )
         return "\n".join(wp_config)
 
+    def _container(self):
+        """Get the WordPress workload container"""
+        return self.unit.get_container("wordpress")
```

Comment:
It might be useful defining both the container name and the service name in a constant

Reply:
Fixed. thanks!